.expeditor/verify.pipeline.yml

```diff
@@ -11,45 +11,45 @@ expeditor:
 
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+- label: run-lint-and-specs-ruby-2.5
```

Comment:
Make sure to also drop ruby 2.4 in the gemspec

Reply:
Are we ready to drop ruby 2.4? Or do we need to pin in in places if it is beings used elsewhere?

Reply:
We are ready to drop 2.4